provisioners/jenkins/jenkinsfiles/aem-opencloud/manage-environments/aws/create-consolidated

```diff
@@ -41,6 +41,7 @@ pipeline {
         stage('Resolve dependencies') {
             steps {
                     JenkinsStagePreStep this
+                    ExecCommand(this, env.TMPDIR, "aem-aws-stack-builder", "mv aem-aws-stack-builder-master/* ./ ")
```

Comment:
This would break versioned releases and it also doesn't cover all repos.
The change described on this ticket https://github.com/shinesolutions/aem-opencloud-manager/issues/14 is preferred.